pkg/router/template/router.go

```diff
@@ -1369,10 +1369,16 @@ func (r *templateRouter) getActiveEndpoints(serviceUnits map[ServiceUnitKey]int3
 func (r *templateRouter) calculateServiceWeights(serviceUnits map[ServiceUnitKey]int32, port string) map[ServiceUnitKey]int32 {
 	serviceUnitNames := make(map[ServiceUnitKey]int32)
 
-	// If there is only 1 service unit, then always set the weight 1
+	// If there is only 1 active service unit, then always set the weight 1
 	// for all the endpoints, except when the service weight is 0.
 	// Scaling the weight to 256 is redundant and causes haproxy to allocate more memory on startup.
```

Comment:
If we don't actually do this scaling, can you update the pre-signature comment that says "The largest weight per endpoint is scaled to 256 to permit better precision results." ?

Reply:
Sure, I added details to that comment regarding having 1 active endpoint.

Reply:
So, my concern was just that these two comments don't really agree with each other.
// If there is more than one active service, the largest weight per endpoint is scaled to 256 to permit better precision results.
// Scaling the weight to 256 is redundant and causes haproxy to allocate more memory on startup.

Reply:
Just changing this might do the trick?
// Scaling the weight to 256 **in this case** is redundant and causes haproxy to allocate more memory on startup.